pyvista/_vtk.py

```diff
@@ -733,6 +739,10 @@ def __init__(self):  # pragma: no cover
 
             raise VTKVersionError('vtkHDRReader requires VTK v9 or newer')
 
+    def lazy_vtkXdmfReader():
+        """Lazy import of the vtkXdmfReader."""
```

Comment:
Need to import `VTKVersionError`, I believe this is the reason for the failing test.
```suggestion
        """Lazy import of the vtkXdmfReader."""
        from pyvista.core.errors import VTKVersionError
```

Reply:
That's right, but don't worry, VTK 8.1.2 will be dropped soon 😉 
https://github.com/pyvista/pyvista/pull/4018